### ISSUE TITLE
gunc download_db: gunc: error: the following arguments are required: -d/--database_file

### ISSUE THREAD

**Issue by uloeber** [grp-bork/gunc#6]
Dear @fullama , 
I get the error 
`gunc download_db ./
usage: gunc [-h] -d  (-i  | -g ) [-p] [-t] [-o] [-s] [-v]
gunc: error: the following arguments are required: -d/--database_file`
In both cases, running 
`gunc download_db ./`
or
`gunc --download_db ./`
What am I doing wrong?
Bests,
Ulrike

**Comment by fullama** [comment]
As far as I can tell you are doing nothing wrong, the same command you put there works fine for me:
![image](https://user-images.githubusercontent.com/8556960/103482446-45440c80-4de1-11eb-9d8f-988de03fdace.jpeg)

Could you maybe try copy paste this command: 

    gunc download_db ./ 

Just to see what happens and maybe post a screen shot of exactly what the output is?

**Comment by uloeber** [comment]
Dear @fullama ,
sur!
![image](https://user-images.githubusercontent.com/4630715/103482837-660d6180-4de3-11eb-98a7-849437b1f192.png)

**Comment by fullama** [comment]
Is it version 1.0.1? 
    
    gunc -v

**Comment by uloeber** [comment]
Installed via conda:
gunc -v
0.1.2

**Comment by fullama** [comment]
Ah.. that version is no longer supported.. (I'll have to see if I can remove it) can you try force conda to install the latest version?

    conda install -n gunc -c bioconda gunc=1.0.1

I'll update the Installation documentation to specify 1.0.1 also..

**Comment by uloeber** [comment]
Thanks!
there seem to be a lot of conflicts. Is this due to the conda receipt? May I need to specify all dependency versions on my own then?
Thanks for your quick help
![image](https://user-images.githubusercontent.com/4630715/103483878-91e01580-4dea-11eb-8416-ee1001264427.png)

**Comment by fullama** [comment]
Issue #5 was a very similar issue: in that case it was the order of the channels that fixed it maybe that might solve your issue here too? 

https://github.com/grp-bork/gunc/issues/5#issuecomment-750512031

**Comment by uloeber** [comment]
Tried to solve it as suggested in #5 , but failed
Then tried it with 
`conda create -n gunc -c bioconda diamond=2.0.4 prodigal python gunc=1.0.1` with the same result
![image](https://user-images.githubusercontent.com/4630715/103484386-c05fef80-4dee-11eb-87c7-800936bd891c.png)

**Comment by fullama** [comment]
Sorry I suddenly stopped answering.. I had to go hospital like 20 mins after my last comment.. did you check the channel order? Is it set up exactly as it is in the link? That's my current suspicion of what it could be?

**Comment by uloeber** [comment]
Dear Anthony,
I hope you are doing better. 
I checked the channel order and am still getting the same error. :(

**Comment by fullama** [comment]
Huh.. are you able to install just diamond by itself? Maybe try install one by one until only gunc is left and see if everything else will install before gunc at the end? It can be very hard to diagnose conda installation issues, maybe this might narrow it down..

**Comment by uloeber** [comment]
pip3 instead of pip made it. thanks!

**Comment by uloeber** [comment]
Well, unfortunately running into the next problem:
![image](https://user-images.githubusercontent.com/4630715/104131894-cc612980-5379-11eb-92b9-6c889edc34b5.png)

**Comment by uloeber** [comment]
`conda create -n gunc -c bioconda -c conda-forge diamond=2.0.4 prodigal python=3.9 gunc=1.0.1 libgcc-ng=9.1.0 libstdcxx-ng=9.3.0`
works!
Still the db parameter seems to not be correctly recognized:
![image](https://user-images.githubusercontent.com/4630715/104132055-362e0300-537b-11eb-9ac6-d33f7915263b.png)

**Comment by fullama** [comment]
Great that we finally got there..! The last error is my fault for forgetting to update that part of the documentation.. it was supposed to be -r (for reference) not -d anymore.. I've updated it now so hopefully no one else will get tripped up by that.. thanks for noticing that though!

Im not sure why conda wouldn't work nicely for you.. I'll look into it but at least it seems to be installed now for you?

Is it working ok for you now or is there anything else I can help with?
